lib/SiteConfig.py

```diff
@@ -243,22 +242,21 @@ def organize_merge(self):
 
                 recipe_file = os.path.join(clone_loc, RECIPE_VARS)
                 if os.path.exists(recipe_file):
-                    msg = f"""Neither --recipe-vars nor --bootprep-config-dir
-                    were specified, so {RECIPE_VARS}  will pulled from the
-                    branch {co_branch} of the {repo} git repo."""
+                    msg = ("Neither --recipe-vars nor --bootprep-config-dir were specified, "
```

Comment:
I'm having second thoughts about trying to use VCS for the bootprep stuff in the Alice timeframe.  I wouldn't be unhappy if we simply commented out all the VCS SW Recipe stuff.

Reply:
FWIW the way I wrote the docs was to tell the user to clone the recipe repo into the admin directory and then continue on.  TBH I was also a bit confused why iuf was doing this step...

Reply:
We can certainly talk about removing it, all I did was fix the formatting since those messages were coming out like
`INFO Neither --recipe-vars nor --bootprep-config-dir                     were
specified, so product_vars.yaml  will pulled from the
branch cray/hpc-csm-software-recipe/22.11.0 of the hpc-csm-software-recipe git
repo.`